desktop/app/README.md

```diff
@@ -0,0 +1,57 @@
+# Mintter Desktop App
+
+This is the desktop app wrapper of the Mintter App. To learn more about the
+folder structure see https://tauri.app.
+
+## Building
```

Comment:
what about launching the app with flags for the daemon?
something like`./dev run-desktop --grpc-port 56002`

Reply:
Good catch! Updated the readme with that info!